pkgs/development/python-modules/mayim/default.nix

```
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+
+, psycopg
+, aiosqlite
+, asyncmy
+
+# test
+, coverage
+, pytest
+, pytest-cov
+, pytest-asyncio
+, tox
+
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "mayim";
+  version = "1.1.0";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```
Could you please use a parameter, `pyproject`, instead?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters